internal/cmd/cmd.go

```diff
@@ -24,14 +24,12 @@ func (v verbose) BeforeApply() error {
 
 type Cmd struct {
 	Local   local.Cmd   `cmd:"" help:"Manage the local Airbyte installation."`
+	Images  images.Cmd  `cmd:"" help:"Manage images used by Airbyte and abctl."`
 	Version version.Cmd `cmd:"" help:"Display version information."`
 	Verbose verbose     `short:"v" help:"Enable verbose output."`
 }
 
 func (c *Cmd) BeforeApply(ctx *kong.Context) error {
-	if _, envVarDNT := os.LookupEnv("DO_NOT_TRACK"); envVarDNT {
-		pterm.Info.Println("Telemetry collection disabled (DO_NOT_TRACK)")
-	}
```

Comment:
This moved to the `local` command, so that it doesn't print at the top of commands like `images manifest` which aren't related to telemetry.